ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dot1x.j2

```diff
@@ -25,8 +25,8 @@
 
 ### 802.1X Interfaces
 
-| Interface | PAE Mode | State | Phone Force Authorized | Reauthentication | Auth Failure Action | Host Mode | Mac Based Auth |
-| --------- | -------- | ------| ---------------------- | ---------------- | ------------------- | --------- | -------------- |
+| Interface | PAE Mode | State | Phone Force Authorized | Reauthentication | Auth Failure Action | Host Mode | Mac Based Auth | Eapol | Auth Failure Fallback MBA |
```

Comment:
Do we really need two columns for one option? Maybe it could be one column Eapol with either disabled of auth failure fallback MBA ?

Reply:
Fixed